### PR TITLE
Lightflow

### PR DESCRIPTION
**Pull request by jrmlhermitte**
Lightflow has quite a few deps. Let's start it off on qas and iss only first.

dep tree installed (pushed [here](https://anaconda.org/jrmlhermitte/repo) first)
1. lightflow
    1. pymongo
    2. celery
        1. case
        2. kombu
            1. amqp
                1. vine
        3. billiard
    3. redis-py
    4. ruamel.yaml
    5. colorlog

### CODE REVIEW

File: recipes-tag/pymongo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set version = "3.6.1" %}
```
Comment (tacaswell) [inline recipes-tag/pymongo/meta.yaml]: we don't need pymongo, it is the deaults channel .

Comment (tacaswell) [reply recipes-tag/pymongo/meta.yaml]: unless we need better than 3.4.0

File: recipes-tag/redis-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% set name = "redis-py" %}
```
Comment (tacaswell) [inline recipes-tag/redis-py/meta.yaml]: We already have this on the anaconda channel.

File: recipes-tag/pymongo/meta.yaml (hunk 1 above)
Comment (jrmlhermitte) [reply recipes-tag/pymongo/meta.yaml]: unfortunately, it's set by the developer [here](https://github.com/AustralianSynchrotron/lightflow/blob/master/setup.py#L47).
We need pymongo 3.6.0

I can ask to see if he can downgrade this requirement if adding this is a problem. It's very likely he doesn't need 3.6.

Comment (tacaswell) [reply recipes-tag/pymongo/meta.yaml]: fair enough.

File: recipes-tag/redis-py/meta.yaml (hunk 2 above)
Comment (jrmlhermitte) [reply recipes-tag/redis-py/meta.yaml]: not sure how i missed it (might have forgotten `-c anaconda` when manually building for lightflow the first time around). just pushed the changes.
I also searched for all these packages on the anaconda channel : `anaconda.org/anaconda` and did not find others. good catch!

**Pull request merged**